src/tasks/DockerPseudoterminal.ts

```diff
@@ -120,10 +120,37 @@ export class DockerPseudoterminal implements Pseudoterminal {
             }
         );
     }
+
+    public async executeCommandInTerminal(command: string, options?: ExecuteCommandInTerminalOptions): Promise<ExecAsyncOutput> {

```

Comment:
Consider naming this execAsyncInTerminal or similar; using Command has to much potential for confusion with CommandProvider/CommandResponse.

Reply:
That makes sense, done!